styles.css

```diff
@@ -780,7 +780,7 @@ footer {
   .footer_icon {
     display: flex;
```

Comment:
- [ ] In your form tag, you included this attribute `novalidate`. Kindly remove it because the HTML validations you have specified on each form input will not work. Remember that part of the requirement is that the form should not submit data if certain fields/form inputs are not filled, thus the request to put the `required` attribute on all form inputs. Kindly make this change. 👍

Reply:
Thank you, it is done